lib/pages/people/view/person_view.dart

```diff
@@ -120,3 +35,97 @@ class PersonView extends StatelessWidget {
     );
   }
 }
+
+class _PersonViewState extends State<PersonView> {
+  final ScrollController _controller = ScrollController();
+  
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Container(
+        color: Theme.of(context).scaffoldBackgroundColor,
+        // TODO(RavanRotaru): extract scrollbar in another widget
+        child: Scrollbar(
```

Comment:
Why is this necessary?

Reply:
We are experimenting on a always shown scroll bar (with functionality suitable web, not mobile), which should be a Widget on it's own whoes child will be passed as a parameter

Reply:
This has nothing to do with the PersonView per se though, right? Can't we place the scrollbar in the Scaffold or something?

And also, what's the reasoning for wanting to always have a scrollbar displayed?

Reply:
Yep, it has nothing directly to do with PersonView. We are moving it in a separate Widget (maybe a Scafold).
The comment is only for me to remember moving it.

We discussed its necessity in the meeting, but, for future reference: The Flutter version we use has Scrollbar widgets designed for mobile usage. I might be wrong or Flutter 2.0 might have different scrollables, but as [this discussion](https://github.com/flutter/flutter/issues/40952) point out this problem won't be addressed soon (a solution is in [this comment](https://github.com/flutter/flutter/issues/40952#issuecomment-733155645) from the same thread)

Reply:
removed the scrollbar for the moment https://github.com/student-hub/acs-upb-mobile/pull/252/commits/060eb2fe8ef8bfa485eb397abde61ef08b9c292f